OpenTabletDriver/Vendors/Wacom/WacomTouchReport.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Numerics;
+using OpenTabletDriver.Plugin.Tablet;
+using OpenTabletDriver.Plugin.Tablet.Touch;
+
+namespace OpenTabletDriver.Vendors.Wacom
+{
+    public struct WacomTouchReport : ITouchReport, IAuxReport
+    {
+        public WacomTouchReport(byte[] report)
+        {
+            Raw = report;
+            AuxButtons = prevAuxButtons ?? new bool[4] { false, false, false, false };
+            Touches = PrevTouches ?? new TouchPoint[maxPoints];
+            if (report[2] == 0x81)
+            {
+                ApplyTouchMask((ushort)(Raw[3] | (Raw[4] << 8)));
+                PrevTouches = (TouchPoint[])Touches.Clone();
+                return;
+            }
+
+            var nChunks = Raw[1];
+            for (var i = 0; i < nChunks; i++)
+            {
+                var offset = i << 3;
+                var touchID = Raw[2 + offset];
+                if (touchID == 0x80)
+                {
+                    AuxButtons = new bool[]
+                    {
+                        (report[3 + offset] & (1 << 0)) != 0,
+                        (report[3 + offset] & (1 << 1)) != 0,
+                        (report[3 + offset] & (1 << 2)) != 0,
+                        (report[3 + offset] & (1 << 3)) != 0
```

Comment:
If the suggestions above are applied, this should also change to:
```suggestion
                        (report[1 + offset] & (1 << 0)) != 0,
                        (report[1 + offset] & (1 << 1)) != 0,
                        (report[1 + offset] & (1 << 2)) != 0,
                        (report[1 + offset] & (1 << 3)) != 0
```